src/index.md

```diff
@@ -1547,7 +1529,63 @@ The Skara tooling includes support for backports. [The official Skara documentat
+
+The release note itself is written in a JBS sub-task to the issue that is used to push the change. There are a few steps to follow in order for the release note to find its way from JBS to the actual release note document.
+
+#. Add the label `release-note=yes` on the main JBS issue you want a release note for. That is, the JBS issue that is used to push the change, **not** the CSR (if there is one).
+#. Create a sub-task of the same JBS issue that you want a release note for (not of the CSR).
+   * Prefix the name of the sub-task with "Release Note:".
+   * The title of the release note sub-task should be a one sentence synopsis that is informative (and concise) enough to attract the attention of users, developers, and maintainers who might be impacted by the change.
+   * Add the `release-note` label to the sub-task.
+   * Add the proper `RN-`label (see below) to the release note sub-task.
+   * Assign the sub-task to the person who should write the release note.
+   * Set the Fix Version of the sub-task to the same value that the main issue has.
+   * Enter the text of the release note in the **Description** field. Some markdown formatting is available (even though it's not rendered in JBS).
+#. Ask your Reviewers to have a look at the release note.
+#. When you are done, Resolve the release note sub-task as "Delivered".
+
+## RN-labels
+
+The RN-label is used to indicate what kind of change the release note is for. All release note sub-tasks must have at least one of these labels.
+
+[**`RN-NewFeature`**]{#RN-NewFeature}
+:   New Feature or enhancement.
+
+[**`RN-IssueFixed`**]{#RN-IssueFixed}
+:   A significant issue which has been fixed, would normally be a regression or an issue which unknowingly released in a new feature.
+
+[**`RN-KnownIssue`**]{#RN-KnownIssue}
+:   An issue that wasn't possible to fix by the time the release was GA'd.
+
+[**`RN-Removed`**]{#RN-Removed}
+:   Covers an API, feature, tool etc. which has been removed from the JDK.
+
+[**`RN-Deprecated`**]{#RN-Deprecated}
+:   Covers an API that has been marked as deprecated in the release.
+
+[**`RN-Important`**]{#RN-Important}
+:   Used to indicate that the release note should be highlighted in some fashion, such as listing it at the beginning of the release notes.
+
+[**`RN-`(distro)**]{#RN-distro}
+:   Used to indicate that the release note is only relevant for a specific JDK distribution. E.g. RN-Oracle
+
+[~~**`RN-Change`**~~]{#RN-Change}
+:   Deprecated.
+
+Also see [release-note](#release-note).
+
```

Comment:
It might be worth pointing to an example of what is considered a good release note in JBS

Reply:
I'd be happy to do that. Do you have any examples of good release notes?

Reply:
Perhaps a specific issue from the 18 notes:  https://jdk.java.net/18/release-notes (note that this link may be transient, but this JBS issues themselves are stable). The 17 release notes may also contain possibilities:  https://www.oracle.com/java/technologies/javase/17-relnote-issues.html .

Reply:
I've added a link to the JDK Release Note index. Hopefully they are good enough to serve as examples.